GNUmakefile

```diff
@@ -98,7 +98,7 @@ testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout $(TESTTIMEOUT) -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
 
 acctests: fmtcheck
-	TF_ACC=1 go test -v ./azurerm/internal/services/$(SERVICE)/tests/ $(TESTARGS) -timeout $(TESTTIMEOUT) -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
+	TF_ACC=1 go test -v ./azurerm/internal/services/$(SERVICE)/ $(TESTARGS) -timeout $(TESTTIMEOUT) -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
```

Comment:
Could we keep this as it was? The *tests* subdirectory doesn't exist for datafactory subpacakge now, whilst it should have.
Maybe can we do a folder layout refactor for the datafactory to introduce the *tests* subdirectory, as other services?

Reply:
Hi @magodo you might want to check with @katbyte . Data factory originally had a tests folder. Someone removed it. I asked the slack channel and was told there is a slow refactor happening to remove the tests folder. I'm fine either way it just needs to be consistent as the acceptance tests are broken when I try to run them.

Reply:
@magodo I'ved reverted to included the tests folder path, its just something I'll always have to change until somebody updates all the services to consistently use (or not) a 'tests' subfolder.

Reply:
You are right (I recall it now), the *tests* subdirectory IS to be deprecated. Whilst that is not happening for all the services, we shall just keep the Makefile as it was for now.